styles.css

```diff
@@ -1,17 +1,31 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
 
-body {
+* {
   margin: 0;
   padding: 0;
 }
```

Comment:
- [ ] Kindly, add the `overflow-x:hidden` to remove the scroll left of the page.

![port32](https://user-images.githubusercontent.com/55635977/130134433-77650143-be7f-48e0-9382-6f3473217214.png)

Reply:
what do you mean ?

Reply:
I am totally confused right now

Reply:
If you are available, I can help you @den-cloud

Reply:
Please do

Reply:
Just add `overflow-x: hidden` in the body, it will prevent the scroll left for the page.

Reply:
thanks